linkml/generators/pydanticgen/templates/attribute.py.jinja

```diff
@@ -7,6 +7,12 @@
     {%- if minimum_value != None %}, ge={{minimum_value}}{% endif -%}
     {%- if maximum_value != None %}, le={{maximum_value}}{% endif -%}
 {%- endif -%}
+{%- if multivalued and exact_cardinality != None %}
+, min_length={{exact_cardinality}}, max_length={{exact_cardinality}}
+{%- elif multivalued %}
```

Comment:
If properly indented, the indentation would cause extra space in the generated Python code

Reply:
See those `-` signs in the `{%-` tags? Thats how jinja manages whitespace while keeping the template readable :) 
https://jinja.palletsprojects.com/en/3.1.x/templates/#whitespace-control

Reply:
Fixed, thank you for that!